lib/kubeclient.rb

```diff
@@ -22,8 +22,12 @@ class Client
       [Kubeclient.const_set(et, Class.new(RecursiveOpenStruct)), et]
     end
 
-    def initialize(uri, version)
+    def initialize(uri, version = 'v1beta3')
       @api_endpoint = (uri.is_a? URI) ? uri : URI.parse(uri)
+
+      unless @api_endpoint.path.include? 'api'
+        @api_endpoint.merge!(File.join(@api_endpoint.path, 'api'))
```

Comment:
> Actually since you have a URI object, perhaps you can just use it, which would be the most accurate.

what do you mean?
user could just pass host:port as part of the uri, without "/api" and without the version.
in the case "/api" and version weren't passed, they need to be appended to the path in uri, no?